clustermesh/clustermesh.go

```diff
@@ -649,7 +649,44 @@ func (ai *accessInformation) validate() bool {
 		ai.ClusterID != "0"
 }
 
-func (k *K8sClusterMesh) extractAccessInformation(ctx context.Context, client k8sClusterMeshImplementation, endpoints []string, verbose bool) (*accessInformation, error) {
+func hasDeprecatedName(secretName string) bool {
```

Comment:
nit: On the first reading I thought this would answer the question "is this a deprecated name", while the semantics is "is there a deprecated name we should try if getting the secret with the given name failed". As such, this would be more maintainable if this function returned the deprecated name itself, instead of a boolean, and an empty string if there is no deprecated name. If so, this function could be called `getDeprecatedName()`.

Reply:
@jrajahalme Thanks for your feedback. I have made changes accordingly.